$MirOS$

XXX hand-edited, due to RCS ID in patch

--- cmd/cannacheck/ccheck.man.orig	Thu Oct  7 11:52:14 2004
+++ cmd/cannacheck/ccheck.man	Thu Oct  7 11:52:16 2004
@@ -19,3 +19,3 @@
 .\" OTHER TORTUOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR
 .\" PERFORMANCE OF THIS SOFTWARE.
-
+.\"
